test/e2e/util.go

```diff
@@ -2098,7 +2098,7 @@ func NewHostExecPodSpec(ns, name string) *api.Pod {
 	pod := &api.Pod{
 		TypeMeta: unversioned.TypeMeta{
 			Kind:       "Pod",
-			APIVersion: latest.GroupOrDie("").GroupVersion.Version,
```

Comment:
I would be interested in seeing that refactor first. I'm OK with something like `latest.GroupOrDie(mygrouppackage.Group)`, as long as the latest package remains singular.

> The old code did exactly what this code is doing now. You simply couldn't tell that it was doing that.

Actually it was the new code I misunderstood, because I hadn't realized you'd duplicated the latest package :)